src/moin/converter/creole_in.py

```diff
@@ -562,7 +562,7 @@ def tablerow_cell_repl(self, stack, cell, cell_text, cell_head=None):
             attrib[moin_page.class_] = 'moin-thead'
         element = moin_page.table_cell(attrib=attrib)
         stack.push(element)
-        self.parse_inline(cell_text, stack)
+        self.parse_inline(cell_text.strip(' '), stack)
```

Comment:
hmm, just `.strip()` would have stripped all whitespace. was it intended to only strip blanks?